### PR TITLE
Wrap Kasocki with service-runner

### PR DESCRIPTION
**Pull request by ottomata**

### CODE REVIEW

File: .jshintrc [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+    "predef": [
+        "ve",
```
Comment (gwicke) [inline .jshintrc]: This is a pretty old version that we stole from VE. Hence the "ve" vestige, here ;)

A newer version is available at https://github.com/gwicke/node-serviceworker-proxy/blob/master/.jshintrc.

File: app.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env node
+'use strict';
+
+const http      = require('http');
+const socketio  = require('socket.io');
+const Kasocki   = require('kasocki');
+const P         = require('bluebird');
```
Comment (gwicke) [inline app.js:7]: Minor: Most style guides discourage this alignment (as it's hard to maintain), but that's a personal preference we don't enforce.

**Comment by gwicke** [comment]
LGTM overall. You might still want to add a trivial server.js in the top level, similar to https://github.com/wikimedia/restbase/blob/master/server.js. This makes it easier to pass commandline parameters to the service (such as -c <config> and -n <workers>) with the old npm version available in Debian. Only newer versions have support for `-- -service -arg`.

**Review by gwicke — CHANGES_REQUESTED** [review]

File: app.js (hunk 2 above)
Comment (ottomata) [reply app.js:7]: Cool, I like maintaining it :D  unless you decided to enforce it, I'll leave it as is.

**Pull request merged**